Content.Shared/Preferences/HumanoidCharacterProfile.cs

```diff
@@ -272,7 +272,7 @@ public HumanoidCharacterProfile WithAntagPreference(string antagId, bool pref)
         }
 
         public string Summary =>
-            Loc.GetString("{0}, {1} years old human. {2:Their} pronouns are {2:they}/{2:them}.", Name, Age, this);
+            Loc.GetString(" This is {0}. {3:They} {3:are} a {2} year old {1}.", Name, Sex.ToString().ToLower(), Age, this);
```

Comment:
```suggestion
             Loc.GetString(" This is {0}. {2:They} {2:are} {1} years old.", Name, Age, this); 
```
Sex already shows in the character sprite so it shouldn't be declared like this tbh, the rest are things that you cant deduct by looking at the sprite so its fine

Reply:
Personally think sex should be up there as it can be hard to tell a lot of the time. Spessman are 32x32 pixels and guys/girls can have different hair - jumpsuits - etc.

In the interest of getting this PR merged i'll commit the suggestion.